### ISSUE TITLE
Unable to reproduce results with Impala

### ISSUE THREAD

**Issue by xuehy** [google-research/football#160]
I use impala to train for the `11_vs_11_easy_stochastic` task with checkpoints reward, and follow the hyper-parameter settings in the paper. The only difference is that I use only 128 actors. The initial reward is about -2. It then goes up to 0 and then oscillates around 0 for a long time. If I use PPO, the reward can keep increasing beyond 0.

 I wonder how many training iterations impala takes to achieve the results reported in the paper?

 Besides, the papers says the batch size is 128, is it the sample count or the number of trajectories?

**Comment by AntonRaichuk** [comment]
"Figure 4" in the paper shows that IMPALA @20M had reward < 0 and @500M had reward > 10. So crossing the zero happens somewhere in between.
While PPO crosses zero much earlier and @20M it already has reward > 5.
Could it be that you just haven't ran it for long enough?

Batch size if 128 of unrolls (which is 32). So each batch has 128*32 transitions.
